spec.emu

```diff
@@ -502,54 +1375,158 @@ contributors: Luca Casonato, Guy Bedford, Nicolò Ribaudo
                 1. Append _module_ to _state_.[[Visited]].
-                1. Let _requestedModulesCount_ be the length of _module_.[[RequestedModules]].
+                1. Let _requestedModulesCount_ be the number of elements in _module_.[[RequestedModules]].
                 1. Set _state_.[[PendingModulesCount]] to _state_.[[PendingModulesCount]] + _requestedModulesCount_.
-                1. For each String _required_ of _module_.[[RequestedModules]], do
-                  1. If _module_.[[LoadedModules]] contains a Record _record_ whose [[Specifier]] is _required_, then
-                    1. Perform InnerModuleLoading(_state_, _record_.[[Module]], <ins>_module_, _required_</ins>).
+                1. <del>For each String _required_ of _module_.[[RequestedModules]], do</del>
+                1. <ins>For each ModuleRequest Record _required_ of _module_.[[RequestedModules]], do</ins>
+                  1. <ins>Let _specifier_ be _required_.[[Specifier]].</ins>
+                  1. If _module_.[[LoadedModules]] contains a Record whose [[Specifier]] is <del>_required_</del><ins>_specifier_</ins>, then
+                    1. Let _record_ be that Record.
+                    1. Perform InnerModuleLoading(_state_, _record_.[[Module]]).
                   1. Else,
-                    1. Perform HostLoadImportedModule(_module_, _required_, _state_.[[HostDefined]], _state_).
+                    1. Perform HostLoadImportedModule(_module_, <del>_required_</del><ins>_specifier_</ins>, _state_.[[HostDefined]], _state_).
                     1. NOTE: HostLoadImportedModule will call FinishLoadingImportedModule, which re-enters the graph loading process through ContinueModuleLoading.
                   1. If _state_.[[IsLoading]] is *false*, return ~unused~.
-              1. Assert: _state_.[[PendingModulesCount]] &ge; 1.
+              1. Assert: _state_.[[PendingModulesCount]] ≥ 1.
               1. Set _state_.[[PendingModulesCount]] to _state_.[[PendingModulesCount]] - 1.
               1. If _state_.[[PendingModulesCount]] = 0, then
                 1. Set _state_.[[IsLoading]] to *false*.
-                1. For each Cyclic Module Record _loaded_ in _state_.[[Visited]], do
+                1. For each Cyclic Module Record _loaded_ of _state_.[[Visited]], do
                   1. If _loaded_.[[Status]] is ~new~, set _loaded_.[[Status]] to ~unlinked~.
-                1. Perform ! Call(_state_.[[PromiseCapability]].[[Resolve]], *undefined*, &laquo; *undefined* &raquo;).
+                1. Perform ! Call(_state_.[[PromiseCapability]].[[Resolve]], *undefined*, « *undefined* »).
               1. Return ~unused~.
             </emu-alg>
           </emu-clause>
+        </emu-clause>
+
+        <emu-clause id="sec-moduledeclarationlinking" type="concrete method" oldids="sec-moduledeclarationinstantiation">
+          <h1>Link ( ): either a normal completion containing ~unused~ or a throw completion</h1>
```

Comment:
You need to add the `<dl>` header to get proper rendering of the return type. Also in `Evaluate`.